src/test/java/be/syntra/devshop/DevshopBack/testutilities/ProductUtils.java

```diff
@@ -89,4 +90,8 @@ public static Product createProductWithId() {
         productList.add(product2);
         return productList;
     }
+
```

Comment:
createProductDtoList() and createDummyArchivedProductList() aren't used anymore and can be removed from this class

Reply:
removed